numba/cuda/cudadrv/driver.py

```diff
@@ -633,8 +976,17 @@ def __len__(self):
         return len(self._cons)
 
 
-_MemoryInfo = namedtuple("_MemoryInfo", "free,total")
+MemoryInfo = namedtuple("MemoryInfo", "free,total")
+"""Free and total memory for a device.
```

Comment:
I have never realized that this is the convention for docstring on global variables. Sphinx is picking it up.

Reply:
It's the first time I've used it - I found it when I was searching for how to document a `namedtuple`.